README.md

```diff
@@ -14,78 +16,78 @@ nRF Connect supports the following devices:
-2. Restart nRF Connect. The app should now appear in the *Launch app* screen.
-
-# Creating apps
-
-To create your own app, follow the documentation on the [project wiki](https://github.com/NordicSemiconductor/pc-nrfconnect-core/wiki). Have a look at the [RSSI viewer app](https://github.com/NordicSemiconductor/pc-nrfconnect-rssi) to see how a real-world app can be implemented. There is also a [boilerplate app](https://github.com/NordicSemiconductor/pc-nrfconnect-boilerplate) that can be used as a starting point.
-
-# Proxy settings
-
-To configure nRF Connect for use behind a proxy, see [proxy settings](https://github.com/NordicSemiconductor/pc-nrfconnect-core/blob/master/doc/proxy-settings.md).
-
-# Linux udev rules
-
-In order to access Nordic USB devices with correct permissions *udev* rules need to be set up once. For this purpose [nrf-udev](https://github.com/NordicSemiconductor/nrf-udev) repository has been created, follow instructions there.
+2. Restart nRF Connect for Desktop. The app should now appear in the apps list.
 
 # Contributing
 
+## Pull requests
+
 Feel free to submit a pull request. In order to accept your pull request, we need you to sign our Contributor License Agreement (CLA). You will see instructions for doing this after having submitted your first pull request. You only need to sign the CLA once, so if you have already done it for another project in the NordicSemiconductor organization, you are good to go.
 
-# Reporting bugs
+## Reporting bugs
 
 If you find any bugs, or have questions or other feedback in general, please submit a post on the [Nordic DevZone](http://devzone.nordicsemi.no) portal.
 Note that bug reports should describe in sufficient detail how to reproduce the bug.
 
-# Building from source
+# Developing
 
-## Dependencies
+If you either want to program new apps for nRF Connect for Desktop or work on the existing core and apps, you probably want to use the sources to compile and run them locally. You find more in-depth info on this in the [project wiki](https://github.com/NordicSemiconductor/pc-nrfconnect-core/wiki).
 
-To build this project you will need to install the following tools:
+## Prerequisites
 
-* Node.js (>=6.9)
-* npm (>=5.6.0) / yarn (>=1.4.0)
-
-### Compilation of native modules
+To build this project you will need to install the following tools on all plattforms:
 
-The project depends on [pc-ble-driver-js](https://github.com/NordicSemiconductor/pc-ble-driver-js) and [pc-nrfjprog-js](https://github.com/NordicSemiconductor/pc-nrfjprog-js) which are native modules. Pre-compiled binaries for these modules are provided for recent Node.js versions on Windows, macOS, and Linux. However, if binaries do not exist for your platform/Node.js version, then refer to the [pc-ble-driver-js README](https://github.com/NordicSemiconductor/pc-ble-driver-js) which describes requirements for compilation.
+* Node.js (>=6.9)
```

Comment:
I'd suggest to stay away from node version and just mention current LTS, not sure about npm but what comes with nodejs nowadays should be fine.

Reply:
Like @bencefr said, maybe just add a link to Node.js like on this page https://github.com/NordicSemiconductor/pc-nrfconnect-core/wiki/Getting-started